sources/VT100Grid.m

```diff
@@ -623,6 +623,20 @@ - (void)setBackgroundColor:(screen_char_t)bg
     }
 }
 
+- (void)setURLCode:(unsigned short)code
+        inRectFrom:(VT100GridCoord)from
+                to:(VT100GridCoord)to {
+    for (int y = from.y; y <= to.y; y++) {
+        screen_char_t *line = [self screenCharsAtLineNumber:y];
+        for (int x = from.x; x <= to.x; x++) {
+            line[x].urlCode = code;
```

Comment:
@gnachman I'm not sure, but maybe it would make sense to check there is no existing `urlCode` in a given range before changing the url… WDYT?